integrationTests/chainSimulator/bridge/common.go

```diff
@@ -38,6 +39,28 @@ type transferData struct {
 	Args     [][]byte
 }
 
+func setStateBridgeOwner(
```

Comment:
Perhaps simply and more generally 'initOwnerAndSysAccState' ?

Reply:
renamed